client/signup/steps/domains/index.jsx

```diff
@@ -603,6 +638,9 @@ const submitDomainStepSelection = ( suggestion, section ) => {
 	if ( suggestion.isBestAlternative ) {
 		tracksObjects.label = 'best-alternative';
 	}
+	if ( abtest( 'prefillDomainStepValue' ) === 'test' && isBoolean( usedSuggestedDomain ) ) {
```

Comment:
Same as doing `typeof usedSuggestedDomain === "boolean"` if we don't wanna use lodash.

Reply:
Good call, I've replaced the `isBoolean` call on this line and removed the import.